src/auth/handlers/session.handler.ts

```diff
@@ -13,16 +14,17 @@ export class SessionHandler {
     private jwt: JwtService
   ) {}
 
-  async createSession(user: User){
+  async createSession(user: User, parsedUA){
+    const hashed = hashFunction(parsedUA.ua);
     const userSession = await this.sessionRepository
       .createQueryBuilder("session")
       .innerJoinAndSelect("session.user", "user")
       .where("user.id = :id", { id: user.id })
+      .where(`session.device ::jsonb @> \'{"sessionId":${hashed}}\'`)
```

Comment:
if use have id then use don't need use session id

Reply:
I'm using session id to know if the user already have session on his devise. I need sessionId to identify the device, and if user doesn't have that session I'll create another for specific device. Only user Id isn't enough.
Is it ok?